stage2/06-connection-details/01-run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+echo "Installing connection-details and dependencies"
+
+cp files/Pipfile* ${ROOTFS_DIR}/
+
+on_chroot << EOF
+PIPENV_PIPFILE="/Pipfile" pipenv --python "$(which python3)" install --system
+EOF
+
+rm ${ROOTFS_DIR}/Pipfile*
```

Comment:
`rm` + word splitting issue can turn into a disaster quickly 😬.

Reply:
Eek, good spot!

We really need to run shellcheck in GH actions on every shell script to make sure we catch things like this.